Sources/SwiftTerm/Apple/AppleTerminalView.swift

```diff
@@ -135,23 +134,25 @@ extension TerminalView {
             let attributedString = attrStrBuffer [row]
             
             if selection.hasSelectionRange == false {
-                if attributedString.attributeKeys.contains(NSAttributedString.Key.selectionBackgroundColor.rawValue) {
+                // This optimization only works on Mac
+                // if attributedString.attributeKeys.contains(NSAttributedString.Key.selectionBackgroundColor.rawValue) {
+
                     let updatedString = NSMutableAttributedString(attributedString: attributedString)
                     updatedString.removeAttribute(.selectionBackgroundColor)
                     attrStrBuffer [row] = updatedString
-                }
+                //}
             }
             
             if selection.hasSelectionRange == true {
-                if !attributedString.attributeKeys.contains(NSAttributedString.Key.selectionBackgroundColor.rawValue) {
+                // This optimization only works on Mac
+                //if !attributedString.attributeKeys.contains(NSAttributedString.Key.selectionBackgroundColor.rawValue) {
                     let updatedString = NSMutableAttributedString(attributedString: attributedString)
                     updatedString.removeAttribute(.selectionBackgroundColor)
                     updateSelectionAttributesIfNeeded(attributedLine: updatedString, row: row, cols: cols)
                     attrStrBuffer [row] = updatedString
-                }
+                //}
```

Comment:
Same as above, need to understand why this change ended up here - it is likely fine, but not sure that it belongs to the UITextInput support.